tests/validation/tests/v3_api/test_cis_scan.py

```diff
@@ -25,6 +25,9 @@
 from .test_rke_cluster_provisioning import HOST_NAME, \
     POD_SECURITY_POLICY_TEMPLATE, get_cis_rke_config  # NOQA
 
+# Skipping all the tests from this module since CIS 1.5 and CIS 1.4 profile are not supported on latest k8s versions.
```

Comment:
just a nit, can we add the k8s version from which we don't support these cis profiles instead of `latest` k8s version here.

Reply:
Actually test_cis_scan is v1 app install, and i don't think this is supported anymore. Please correct me if I am wrong @rishabhmsra

Reply:
@rishabhmsra if you can confirm if this ^ is the case, we can change the comment based on that.  LGTM otherwise

Reply:
Yes Somwya that's correct.